### DISCUSSION TITLE
Easiest way to get "fee estimates" JSON?

### DISCUSSION THREAD

**Discussion by MegalithicBTC** [romanz/electrs#1063] [Q&A]
Hi, thanks for this great project.

I'm trying to get a list of fee estimates as JSON, compatible with this Esplora endpoint: https://github.com/Blockstream/esplora/blob/master/API.md#get-fee-estimates

I'm looking to produce a list of fees like this: https://mempool.space/api/fee-estimates

My understanding is that Electrs doesn't provide an HTTP api. (Am I wrong?)

I also don't see a way with Electrs to easily output a "compatible" set of fee estimates like this.

So to do this, would the best bet be to iterate with the `blockchain.estimatefee` command, like this?
```
export const getFeeEstimates = async (): Promise<FeeEstimates> => {
  const blockTargets = [1, 2, 3, 6, 12, 24, 144, 504, 1008];
  const feeEstimates: FeeEstimates = {};

  for (const blocks of blockTargets) {
    const feeRate = await jsonRpcRequest<number>('blockchain.estimatefee', [blocks]);
    feeEstimates[blocks] = feeRate;
  }

  return feeEstimates;
}
```
thanks

**Comment by Kixunil** [comment]
> My understanding is that Electrs doesn't provide an HTTP api.

Correct. If you need HTTP you may want to setup a proxy or use a fork.
